speedcord/http.py

```diff
@@ -70,6 +88,11 @@ def __init__(self, token, *, baseuri="https://discord.com/api/v7", loop=asyncio.
         self.retry_attempts = 3
 
     async def create_ws(self, url, *, compression) -> ClientWebSocketResponse:
+        """
+        Opens a websocket to the specified url. Used when creating shards. 
+        :param url: The url that the websocket will conenct to. 
+        :param compression: Whether to enable compression. Currently is not used. Refer to https://discord.com/developers/docs/topics/gateway#encoding-and-compression. 
```

Comment:
This is wrong. This is the compression threshold for the websocket. The compression at that documentation url is zlib which we currently do not support